demos/embeddings/README.md

```diff
@@ -0,0 +1,167 @@
+convert_tokenizer -o models/gte-large-en-v1.5-tokenizer/1 Alibaba-NLP/gte-large-en-v1.5
+optimum-cli export openvino --disable-convert-tokenizer --model Alibaba-NLP/gte-large-en-v1.5 --task feature-extraction --weight-format int8 --trust-remote-code --library sentence_transformers  models/gte-large-en-v1.5-embeddings/1
+```
+> **Note** Change the `--weight-format` to quantize the model to `fp16`, `int8` or `int4` precision to reduce memory consumption and improve performance.
+
+You should have a model folder like below:
+```bash
+tree models/
+models/
+├── graph.pbtxt
+├── gte-large-en-v1.5-embeddings
+│   └── 1
+│       ├── config.json
+│       ├── openvino_model.bin
+│       ├── openvino_model.xml
+│       ├── special_tokens_map.json
+│       ├── tokenizer_config.json
+│       ├── tokenizer.json
+│       └── vocab.txt
+├── gte-large-en-v1.5-tokenizer
+│   └── 1
+│       ├── openvino_tokenizer.bin
+│       └── openvino_tokenizer.xml
+└── subconfig.json
+
+```
+
+The default configuration of the `LLMExecutor` should work in most cases but the parameters can be tuned inside the `node_options` section in the `graph.pbtxt` file. 
+Runtime configuration for both models can be tuned in `subconfig.json` file. 
+
+## Server configuration
+Prepare config.json:
+```bash
+cat config.json
+{
+    "model_config_list": [],
+    "mediapipe_config_list": [
+        {
+            "name": "Alibaba-NLP/gte-large-en-v1.5",
+            "base_path": "models"
+        }
+    ]
+}
+```
+
+
+## Start-up
+```bash
+docker run -d --rm -p 8000:8000 -v $(pwd)/:/workspace:ro openvino/model_server:latest --port 9000 --rest_port 8000 --config_path /workspace/config.json --cpu_extension /ovms/lib/libopenvino_tokenizers.so
+```
+In case you want to use GPU device to run the embeddings model, add extra docker parameters `--device /dev/dri --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1)` 
+to `docker run` command, use the image with GPU support and make sure set the target_device in subconfig.json to GPU. 
+Also make sure the export model quantization level and cache size fit to the GPU memory.
+```
+
+
+Wait for the model to load. You can check the status with a simple command:
+```bash
+curl -s http://localhost:8000/v1/config | jq -c .
+{"Alibaba-NLP/gte-large-en-v1.5":{"model_version_status":[{"version":"1","state":"AVAILABLE","status":{"error_code":"OK","error_message":"OK"}}]},"embeddings_model":{"model_version_status":[{"version":"1","state":"AVAILABLE","status":{"error_code":"OK","error_message":"OK"}}]},"tokenizer":{"model_version_status":[{"version":"1","state":"AVAILABLE","status":{"error_code":"OK","error_message":"OK"}}]}}
+```
+
+## Client code
+
+A single servable exposes both `chat/completions` and `completions` endpoints with and without stream capabilities.
+Chat endpoint is expected to be used for scenarios where conversation context should be pasted by the client and the model prompt is created by the server based on the jinja model template.
```

Comment:
"The chat endpoint is designed for scenarios where the conversation context should be provided by the client, while the model prompt is generated by the server based on the Jinja model template."